ods_ci/tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot

```diff
@@ -54,3 +55,33 @@ Get Query Results
         END
     END
     RETURN  ${metrics_query_result_row1_value}
+
+Run OpenShift Metrics Query
+    [Documentation]    Runs a query in the Monitoring section of Open Shift
+    ...    Note: in order to run this keyword OCP_ADMIN_USER.USERNAME needs to
```

Comment:
Are you planning to do a similar test for non-admin users? Should they be able to watch metrics from their projects as well?

Reply:
Yes, modified this kW to parameterise  user